Makefile

```diff
@@ -69,11 +69,11 @@ reindent:
 
 .PHONY: test
 test:
-	@cd tests; $(PYTHON) run.py --ignore py35 -v $(TEST)
+	@cd tests; $(PYTHON) run.py -v $(TEST)
 
 .PHONY: test-async
 test-async:
-	@cd tests; $(PYTHON) run.py -v $(TEST)
```

Comment:
This is no longer necessary because of the changes I made in `tests/conftest.py`.